### ISSUE TITLE
E418 issues

### ISSUE THREAD

**Issue by TotalVerb** [tonyhffong/Lint.jl#182]
```
julia> lintstr("    a, = (1,2,3)")
0-element Array{Lint.LintMessage,1}

julia> lintstr("    a,b = (1,2,3)")
1-element Array{Lint.LintMessage,1}:
 none:1 E418 Tuple{Int64,Int64,Int64}: RHS is a tuple, 2 of 3 variables used

julia> lintstr("    a,b,c = (1,2,3)")
0-element Array{Lint.LintMessage,1}

julia> lintstr("    a,b,c,d = (1,2,3)")
1-element Array{Lint.LintMessage,1}:
 none:1 E418 Tuple{Int64,Int64,Int64}: RHS is a tuple, 4 of 3 variables used
```

Case 1: this is a likely problem and we should warn, see below.
Case 2: this is a likely problem and we should warn, but E418 is probably too severe. This code runs perfectly well; all the numbers past the first two are discarded.
Case 3: this is fine
Case 4: this is fine
